visualization_and_plotting/readme.md

```diff
@@ -0,0 +1,43 @@
+#Simple horizontal Bar Plots
+---
+#R
+```r
+library(dplyr)
+library(magrittr)
+library(ggplot2)
+
+iris %>%
+    dplyr::group_by(Species) %>%
+    dplyr::summarise(Sepal.Length = mean(Sepal.Length)) %>%
+    ggplot(aes(Species, Sepal.Length)) + geom_bar(stat = 'identity') + 
+    coord_flip() + ggtitle('Sepal Length by Species') + 
+    ggsave('r_sep_length_by_species.png')
+```
+<img src="r_sep_length_by_species.png" width="500" height="350" />
+
+
+#Python
+```python
+import numpy  as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from   sklearn import datasets
```

Comment:
I should take my [own advice](https://github.com/iveksl2/goodcode/blob/master/suggestions.md) .  Maybe use these Fancy linter's